lib/index.js

```diff
@@ -8,6 +8,9 @@ var defaults = require('@ndhoule/defaults');
 var integration = require('@segment/analytics.js-integration');
 var onBody = require('on-body');
 
+/** TO-DO: connect urlChanged to Segment URL-change event **/
+var urlChanged = true;
```

Comment:
Hey @noah-berman - the analytics.js api currently doesn't expose any kind of url change hook. Is there something else that you could reference to test for url changes?

Reply:
One option would be to store the value of `window.location.href` in a class property (called something like `currentUrl`) when `initialize` is called (which should only be when the DOM is refreshed). Then, inside `page`, check to see if `window.location.href` is equal to `this.currentUrl`. If not, then call `virtualPage`. If so, then do the regular page load logic.